dogstatsd.py

```diff
@@ -228,6 +233,31 @@ def submit_events(self, events):
             finally:
                 conn.close()
 
+    def submit_service_checks(self, service_checks):
+        headers = {'Content-Type':'application/json'}
+        method = 'POST'
+
+        params = {}
+        if self.api_key:
+            params['api_key'] = self.api_key
+
+        url = '/api/v1/check_run?{0}'.format(urlencode(params))
+
+        status = None
+        conn = self.http_conn_cls(self.api_host)
+        try:
+            start_time = time()
+            conn.request(method, url, json.dumps(service_checks), headers)
```

Comment:
Could you use the requests library instead ? We recently switched to it in dogstatsd 
https://github.com/DataDog/dd-agent/pull/1290